src/scss/settings/colors/_reserved-color-scale.scss

```diff
@@ -1,4 +1,8 @@
-$level-2-error: rgb(224, 63, 63); // Level 2 - Error
-$level-2-success: rgb(21, 188, 139); // Level 2 - Success
-$level-2-warning: rgb(255, 164, 28); //  Level 2 - Warning
-$level-2-info: rgb(31, 163, 255); // Level 2 - Info
+$success: rgb(21, 188, 139); // Level 2 - Success
```

Comment:
We named them originally as they are on frontier-pay. We don't want to stick with that?

Reply:
Yes, the name should be the same as in frontify, but we still can prefix that name with `$color-`. My problem is that if I see `$success` somewhere in the middle of the code, I don't know that should I imagine by that. Is it a message? A boolean? Nope, it's a color! It might be confusing.